src/ontology/mondo-edit.obo

```diff
@@ -430946,6 +431198,15 @@ xref: ICD10:Q87.3 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:58
 xref: Orphanet:583097 {source="MONDO:equivalentTo"}
 is_a: MONDO:0006574 {source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-4142-7153"} ! lipomatosis
 
+[Term]
+id: MONDO:0035605
+name: B-lymphoblastic leukemia/lymphoma with recurrent genetic abnormality
+subset: ordo_subtype_of_a_disorder
+xref: ICD10:C91.0 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:585877"}
+xref: NCIT:C80328 {source="MONDO:equivalentTo"}
+xref: Orphanet:585877 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004947 {source="https://orcid.org/0000-0001-5208-3432"} ! B-cell acute lymphoblastic leukemia
```

Comment:
From NCIT: The term refers to precursor lymphoid neoplasms which are composed of B-lymphoblasts and characterized by the presence of recurrent genetic abnormalities.

It is unclear whether this is "acute".
Maybe a better parent would be "precursor lymphoblastic lymphoma/leukemia"